content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -39,7 +39,7 @@ is tightly integrated with the tools to prevent common errors and simplify troub
 Every operator implementation requires a
 [custom resource definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
 to define its custom resource, that is, its API. Istio's operator API is defined by the
-[`IstioControlPlane` CRD](/docs/reference/config/istio.operator.v1alpha12.pb/),
+[`IstioControlPlane` CRD](https://github.com/istio/istio.io/blob/release-1.4/content/en/docs/reference/config/istio.operator.v1alpha12.pb/index.html),
```

Comment:
This link takes the user to some HTML wrapped in Github UI.  Is that intended?

Reply:
I think this should point to the 1.4 archive:
```suggestion
[`IstioControlPlane` CRD](https://archive.istio.io/v1.4/docs/reference/config/istio.operator.v1alpha12.pb/),
```